Writerside/topics/EventSource.md

```diff
@@ -0,0 +1,84 @@
+# EventSource
```

Comment:
Does Writerside support Mermaid or something of the like? I think it'd be useful to sketch out the different components the event sources communicates with in a diagram so it's easier to visualize.